src/transformers/tokenization_transfo_xl.py

```diff
@@ -271,6 +271,33 @@ def add_symbol(self, sym):
             self.idx2sym.append(sym)
             self.sym2idx[sym] = len(self.idx2sym) - 1
 
+    def move_added_token(self, sym: str, target_idx: int):
+        """
+        Moves and added token to a specific position in the vocab.
```

Comment:
`Moves added token` without `and` I think

Reply:
Yeah sure, I was going to write `Moves an added token`